lib/tpm2_eventlog_yaml.c

```diff
@@ -402,6 +403,58 @@ static bool yaml_uefi_var(UEFI_VARIABLE_DATA *data, size_t size, UINT32 type,
                 }
                 return true;
             }
+
+            regex_t regex;
+            int r;
+
+            r = regcomp(&regex, "^Boot[0-9a-fA-F]\\{4\\}$", 0);
+            if (r != 0) {
+                free(ret);
+                LOG_ERR("regcomp() returned %d\n", r);
+                return false;
+            }
+
+            r = regexec(&regex, ret, 0, NULL, 0);
```

Comment:
Can't you just:
1. Verify string is length of 10
2. startswith Boot
3. Followed by isalnum()
3. ends with \{4\}

Id rather avoid adding this dependency on GLIBC.

Reply:
OK, that's consistent with the view of @idesai. I'll make that change soon.